metaflow/plugins/datatools/s3/s3op.py

```diff
@@ -826,7 +826,10 @@ def _files():
             # directly. This will not work, for example, if only some lines have
             # an idx specified (in some cases)
             for line in open(filelist, mode="rb"):
-                r = json.loads(line)
+                transformer = lambda x: x
```

Comment:
This looks fine but I would move this much higher (probably around top of file) to avoid paying the cost all the time. The information needed to determine this is static for all intents and purpose so doesn't need to be in the loop. I don't know if there are other instances of this btw. Haven't checked.

Reply:
Will have a look.